server/plugin.go

```diff
@@ -136,11 +136,14 @@ func (p *Plugin) OnActivate() error {
 	}
 	p.mm.Log.Debug("initialized slash commands")
 
-	err = p.proxy.SynchronizeInstalledApps()
-	if err != nil {
-		p.mm.Log.Error("failed to update apps", "err", err.Error())
+	if conf.MattermostCloudMode {
```

Comment:
Does it make sense to also do that in dev mode given that the install command for AWS is available?